README.md

```diff
@@ -194,19 +194,20 @@ specified target's database and downloads it to `tmp/`.
 `geordi dump staging -l` (with a Capistrano deploy target and the `--load`
 option) sources the dump into the development database after downloading it.
 
-If you are using multiple databases per environment, you have to specify the
-database name. To dump a specific database, pass the database name with the
-`-d`/`--database` option. To load a dump into a specific database, pass the
-database name with the `-L`/`--load-database` option.
+If you are using multiple databases per environment, Geordi defaults to the
+"primary" database, or the first entry in database.yml. To target a specific
+database, pass the database name like this: ```
+geordi dump -d  primary
```

Comment:
There's a double space

```suggestion
geordi dump -d primary
```

Please move the beginning of the code block to its own line, not all Markdown interpreters process it properly with an inline beginning.